lib/src/__tests__/DatePickerTestingLib.test.tsx

```diff
@@ -1,15 +1,43 @@
 import * as React from 'react';
+import deLocale from 'date-fns/locale/de';
+import enLocale from 'date-fns/locale/en-US';
 import TextField from '@material-ui/core/TextField';
 import { getByMuiTest } from './test-utils';
-import { screen, waitFor } from '@testing-library/react';
-import { utilsToUse, FakeTransitionComponent } from './test-utils';
+import { MaterialUiPickersDate } from '../typings';
 import { createClientRender, fireEvent } from './createClientRender';
-import { DatePicker, MobileDatePicker, DesktopDatePicker } from '@material-ui/pickers';
+import { render as rtlRender, screen, waitFor } from '@testing-library/react';
```

Comment:
I don't think that using the render method exposed the testing library is an option. What about the strict mode, cleanup, extra props logic?

```suggestion
import { screen, waitFor } from '@testing-library/react';
```

Instead, we can nest two Localization Provider

Reply:
I believe cleanup happens automatically; https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-cleanup.

Reply:
Interesting. @eps1lon do we need to handle the cleanup logic in `createClientRender`?

Reply:
If you're using `jest`, yes. Though I might have an idea why this is problematic. I suggest manually cleaning up (and importing from /pure) so that it's easier to debug cleanup logic until we come up with a robust solution to clean up scheduled tasks.

Reply:
Cool, thanks for the insight!